packages/core/src/tracing/tracingUtil.js

```diff
@@ -176,6 +176,19 @@ exports.shortenDatabaseStatement = function shortenDatabaseStatement(stmt) {
   return stmt.substring(0, 4000);
 };
 
+/**
+ * @param {string} connectionStr
+ * @returns {string}
+ */
+exports.sanitizeConnectionStr = function sanitizeConnectionStr(connectionStr) {
+  if (connectionStr == null || typeof connectionStr !== 'string') {
+    return undefined;
+  }
+
+  const replaced = connectionStr.replace(/PWD=.*?(?=;)/, 'PWD=REPLACED');
```

Comment:
_praise:_ Good catch to replace this. When we do secrets scrubbing we usually use `<redacted>` instead of `REPLACED, so we could go with that for consistency.

```suggestion
  const replaced = connectionStr.replace(/PWD=.*?(?=;)/, 'PWD=<redacted>');
```

Reply:
Will change manually 👍

Reply:
Done 👍